docs/core/tutorials/using-on-mac-vs-full-solution.md

```diff
@@ -68,7 +68,7 @@ If you've already installed the prerequisites and Visual Studio for Mac, skip th
 
 Unit tests provide automated software testing during your development and publishing. The testing framework that you use in this tutorial is [xUnit](https://xunit.github.io/).
```

Comment:
there's also a comment on the original issue of calling out the specific version of xunit used